app/lib/server/functions/setUserActiveStatus.js

```diff
@@ -39,6 +39,15 @@ export function setUserActiveStatus(userId, active, confirmRelinquish = false) {
 		return false;
 	}
 
+	const userAdmin = Users.findOneAdmin(userId.count);
```

Comment:
This should only be called if you're actually trying to disable the user, like inside the following if statement